sources/us/ky/boyd.json

```diff
@@ -0,0 +1,23 @@
+{
+    "coverage": {
+        "US Census": {
+            "geoid": "21019",
+            "name": "Boyd County",
+            "state": "Kentucky"
+        },
+        "country": "us",
+        "state": "ky"
+    },
+    "data": "https://rawgit.com/jtroe/b48bf035ce738222d5267ee38e54df8b/raw/094dd554e8550dcfdb6262583d90e4acc6c29c90/ky-boyd.json",
+    "note": "Shapefile converted to GeoJSON via mapshaper.org",
+    "type": "http",
+    "conform": {
+        "type": "geojson",
+        "number": {
+            "field": "NUMBER"
```

Comment:
This should be `"number": "NUMBER",`. No need to specify a `field` in an object.